website/public/locales/en/leaderboard.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Just checking my understanding here.  Does the `leaderboard.tsx` page automatically check this file first for entries?

Reply:
Hey thanks for the review. To explain a little clearer, to load the translations you have to use the `useTranslation` hook and pass the namespace you wish to load. I've been keeping namespaces per page apart from common ones used for page level components. We could use one huge file but I'm not sure if it's the best way forward. Separate namespaces allow quicker loading times along with maintainable json files. I hope that answers your question?